_includes/configuration/valid-styles.md

```diff
@@ -1,6 +1,8 @@
 ## valid_styles
 
-This option enables you specify the available styles for each element. This means you can force ordering and only specific styles will be valid within style attribute values. Specify the element name, then it's styles or `*` for all elements.
+This option allows you to specify the available styles for each element. This option also forces the ordering of the styles. To use this option, specify the element name, then the allowed styles or `*` for all elements.
+
+By default, `valid_styles` is undefined, therefore all styles are valid. If a [`schema`](#schema) is set, `valid_styles` will override relevant parts of the `schema`.
```

Comment:
The default at all times is to allow all styles, the only time that isn't the case is when `valid_styles` or `invalid_styles` is set. It really has nothing to do with the schema, as that only applies when manually instantiating `tinymce.dom.Styles`, but that doesn't apply to this setting.

Reply:
As such I think you can just say something like this instead maybe?

```By default all styles will be allowed, unless `valid_styles` or `invalid_styles` is configured.```

Reply:
fixed